From d9a3d12f0f6a8d555431a6bdf0dbc1f597f942e4 Mon Sep 17 00:00:00 2001
From: Hiroshi Miura <miurahr@linux.com>
Date: Wed, 21 Feb 2018 13:05:26 +0900
Subject: [PATCH 9/9] [cmake] Fix -config.cmake installation

Signed-off-by: Hiroshi Miura <miurahr@linux.com>
---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8793333..8c42907 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,6 +79,8 @@ configure_package_config_file(cmake/project-config.cmake.in ${PROJECT_NAME}-conf
         INSTALL_DESTINATION ${INSTALL_CMAKE_DIR})
 write_basic_package_version_file(${PROJECT_NAME}-config-version.cmake
         VERSION ${PROJECT_VERSION} COMPATIBILITY SameMajorVersion)
-install(FILES ${PROJECT_NAME}-config.cmake ${PROJECT_NAME}-config-version.cmake DESTINATION "${INSTALL_CMAKE_DIR}")
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake
+              ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake
+        DESTINATION "${INSTALL_CMAKE_DIR}")
 install(EXPORT ${PROJECT_NAME}-depends FILE ${PROJECT_NAME}-depends.cmake
         DESTINATION "${INSTALL_CMAKE_DIR}")
-- 
2.16.1

